addons/octahedral_impostors/scripts/impostor_baker.gd

```diff
@@ -334,22 +340,13 @@ func export_packed_scene(pack_path: String) -> void:
 	else:
 		mat.shader = light_shader
 
-	# minimize and maximize the Godot window to trigger importing the images
-	OS.window_minimized = true
-	OS.window_minimized = false
+	if plugin:
+		plugin.get_editor_interface().get_resource_filesystem().scan()
 
 	var albedo_texture: StreamTexture
 	var normal_depth_texture: StreamTexture
 	var orm_texture: StreamTexture
 
-	# wait until the images have all been (re)imported.
-	while not (albedo_texture and normal_depth_texture and (orm_texture if is_standard_shader else true)):
-		albedo_texture = load(pack_path.plus_file(base_filename))
-		normal_depth_texture = load(pack_path.plus_file(normal_depth_filename))
-		if is_standard_shader:
-			orm_texture = load(pack_path.plus_file(orm_filename))
-		yield(get_tree(), "idle_frame")
-
```

Comment:
I think this part was removed by mistake.

Reply:
Actually, I reminded that part on purpose, but I must've been to hasty because I didn't treat if it would work without it. 😅
I'll make the changes tomorrow, as I'm not in my PC currently.

Reply:
It seems that since Godot 3.1, we can eliminate errors in the Godot console by replacing the current while loop with ResourceLoader.exists(path) and then load.